packages/adapter-api/src/element_id.ts

```diff
@@ -119,6 +119,7 @@ export class ElemID {
   readonly typeName: string
   readonly idType: ElemIDType
   private readonly nameParts: ReadonlyArray<string>
+  private fullName: string
```

Comment:
sorry for the late comment, but
```suggestion
  private readonly fullName: string
```